ci/scrape-targets.py

```diff
@@ -0,0 +1,16 @@
+from bs4 import BeautifulSoup
+import requests
+
+url = "https://doc.rust-lang.org/nightly/rustc/platform-support.html"
+body = requests.get(url).text
+soup = BeautifulSoup(body, features="html.parser")
+# The tables are:
+# Tier 1
+# Tier 2 with host tools
+# Tier 2 without host tools <-- we want this one
```

Comment:
Why only tier 2 without host tools, and not also tier 2 with host tools?

Reply:
:thinking: At one point I was thinking that the "with host tools" part of the build would ensure that the sysroot builds, but I suppose if we never test the host tools, they might not build.

Reply:
Also this doesn't imply that the sysroot builds in Miri.